206 msgs wording

ShiriMoran opened the pull request.

adisos commented:
can you also add on this branch, a log message indicating "started synthesis" ?
(already have similar message for other stages, such as "started parsing the given NSX config" , "compute connectivity on parsed config" )

adisos commented:
not sure I understand the message ` NSX expressions' left component is of type *collector.PathExpression which is not supported. (Currently only NSX condition are supported.)`

adisos commented on pkg/symbolicexpr/tagTerm.go:
```diff
@@ -60,8 +60,7 @@ func getTagTermsForCondition(cond *collector.Condition) *tagAtomicTerm {
 		cond.Condition.Operator == nil {
-		logging.Infof("supported nsx condition with type VM, key tag and non empty operator."+
-			"\n\t %+v not supported", *cond)
+		logging.Debugf("NSX condition %v not supported", cond.String())
```
```suggestion
		logging.Debugf("NSX condition %s not supported", cond.String())
```

ShiriMoran commented:
> not sure I understand the message ` NSX expressions' left component is of type *collector.PathExpression which is not supported. (Currently only NSX condition are supported.)`
> 
> and how can we infer to which group this message is related to?
1. Added the group to the msg
2. I rewrote the msg; if it is still not clear we'll have to take this offline
